File: CS/WpfApplication1/XtraReport1.cs
using System;
using System.Drawing;
using System.Collections;
using System.ComponentModel;
using DevExpress.XtraReports.UI;
using System.Windows;

namespace WpfApplication1
{
    public partial class XtraReport1 : DevExpress.XtraReports.UI.XtraReport
    {
        public XtraReport1()
        {
            InitializeComponent();
        }

        private void xrTable1_PreviewClick(object sender, PreviewMouseEventArgs e)
        {
            //MessageBox.Show("Row Clicked!");
        }

    }
}

File: VB/WpfApplication1/XtraReport1.vb
Imports System
Imports System.Drawing
Imports System.Collections
Imports System.ComponentModel
Imports DevExpress.XtraReports.UI
Imports System.Windows

Namespace WpfApplication1
    Partial Public Class XtraReport1
        Inherits DevExpress.XtraReports.UI.XtraReport

        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub xrTable1_PreviewClick(ByVal sender As Object, ByVal e As PreviewMouseEventArgs) Handles xrTable1.PreviewClick
            'MessageBox.Show("Row Clicked!");
        End Sub

    End Class
End Namespace
